spk/syncthing/Makefile

```diff
@@ -12,7 +12,7 @@ MAINTAINER = acolomb
 DESCRIPTION = Automatically sync files via secure, distributed technology.
 DESCRIPTION_FRE = Synchronisation automatique de fichiers via une technologie sécurisée et distribuée.
 DISPLAY_NAME = Syncthing
-CHANGELOG = "Update syncthing to v1.19.0.  Add wizard page to preconfigure credentials for the Web GUI.  Integrate with the certificate management UI in Control Panel."
+CHANGELOG = "Update syncthing to v1.19.0. Add wizard page to preconfigure credentials for the Web GUI. Integrate with the certificate management UI in Control Panel. Fix DSM 7 data migration."
```

Comment:
I routinely add the double spaces after a full-stop btw. because it's an accepted norm (one of several) in English writing.  And my editor (Emacs) treats it automatically when reflowing text.

Reply:
in #5118 I numbered the change log entries, so this gets obsolete.